fm-overlays.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+	Plugin Name: Fieldmanager Overlays
+	Plugin URI: https://github.com/alleyinteractive/fm-overlays
+	Description: A Fieldmanager extension which creates overlays/modal boxes.
```

Comment:
this description doesn't really do it for me. where does it create them? why? with what? you get it. how about...

> A Fieldmanager extension to render dynamic content in modal boxes/overlays on a site.